CHANGELOG.md

```diff
@@ -1,233 +1,312 @@
 # Changelog
+
```

Comment:
:question: What's up with all these spaces? Is that the desired behaviour of updating the changelog?